dev/system-test/firestore.ts

```diff
@@ -444,7 +444,8 @@ describe('DocumentReference class', () => {
   });
 
   it('has set() method', () => {
-    const allSupportedTypesObject = {
+    /* eslint-disable @typescript-eslint/no-explicit-any */
+    const allSupportedTypesObject: {[field: string]: any} = {
```

Comment:
This can probably be `unknown`.

Reply:
We need `any` since we access `allSupportedTypesObject.pathValue.path` on L478.